docs/platform/web-project.md

```diff
@@ -30,72 +30,143 @@ a _feature_ grouping it together with its dependencies,
 and a _KAR_ file, so you don't need to publish your bundle to a remote (or even local) repository and 
 your dependencies are available without the need for a network connection.
 
-To ease the process, a Maven Archetype is provided that lays out the recommended project structure for you.
 
 ## Prerequisites
 
 - Java 1.8
-- Maven >= 3.0.3
+- Maven 3+
+- This [settings.xml](https://github.com/pentaho/maven-parent-poms/blob/master/maven-support-files/settings.xml) 
+in your <user-home>/.m2 directory
+
 
 ## Recommended Maven project directory layout
 
-### Using the Archetype
+Pentaho maven OSGI projects are commonly composed by a [root project](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html) that contains an `impl` 
+folder for bundle implementation [modules](https://maven.apache.org/guides/mini/guide-multiple-modules.html) and an `assemblies` folder for modules concerning the provisioning of those bundles.
 
-```shell
-mvn archetype:generate ...
+If the project contemplates multiple bundles then `impl` will itself include multiple 
+modules, one for each bundle. If the project only has a single bundle then it is OK for `impl` to 
+directly contain the bundle implementation module without the need for a sub module.
+
+A provisioning module, in the context of a Pentaho OSGI project, is a module that automates, 
+as much as possible, the creation of a feature file and potentially the KAR file that allows that feature 
+to be easily deployed. 
+One reason why multiple provisioning modules might exist is that it may desired to deploy the bundle(s) in 
```

Comment:
@dcleao I added the missing "be" and changed to "desirable" as suggested by @carlosrusso